styles.css

```diff
@@ -501,3 +501,82 @@ section .portfolio {
 .about-container {
   display: flex;
 }
+
+/* Contact me section */
+
+#contact-section {
+  background: #6070ff;
+  padding-bottom: 7rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-bottom-left-radius: 4.6875rem;
+  border-top-right-radius: 7.1875rem;
+  background-image: url(/assets/header-bg.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  /* background-position: 20% 40%; */
```

Comment:
- [ ] [REQUIRED] Please, remove this commented code. Commented code makes the code base look messy and unorganized. You can use `git reset <COMMIT_HASH>` to reset back to old code 🔙.

Reply:
Thank you @keroloussamy I have applied this and pushed from VS code!

All the best-K